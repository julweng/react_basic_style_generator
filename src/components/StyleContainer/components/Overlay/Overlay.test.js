import React from "react"
import { render, cleanup } from "@testing-library/react"
import "jest-dom/extend-expect"
import Overlay from "./index"

describe("Overlay", () => {
  afterEach(cleanup)

  it("Should render without crashing", () => {
    const { container } = render(<Overlay />)
    expect(container).toBeTruthy()
    expect(container).toMatchSnapshot()
  })
})
