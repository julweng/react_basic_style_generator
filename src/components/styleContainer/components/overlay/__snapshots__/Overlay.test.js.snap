// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overlay Should render without crashing 1`] = `
<div>
  <div
    class="overlay"
  />
</div>
`;
